taskset.go

```diff
@@ -229,6 +229,22 @@ func (ts *TaskSet) MustGetByID(id int) Task {
 	return *ts.tasksByID[id]
 }
 
+func (ts *TaskSet) SearchForUUID(uuid string) []string {
+	var uuids []string
+	for _,task := range ts.tasks {
+		if strings.HasPrefix(task.UUID,uuid){
+			uuids = append(uuids, task.UUID)
+		}
+	}
+	return uuids
+}
+func (ts *TaskSet) MustGetByUUID(uuid string) Task {
+	if ts.tasksByUUID[uuid] == nil {
+		ExitFail("No task with UUID %v exists.", uuid)
```

Comment:
We should probably return `(Task, error)` here, and call `ExitFail` at a higher level.

Reply:
For testability, yes I agree. I tried to reduce lines + error handling with `ExitAbort` but it's not practical when adding unit tests when called deep within.